src/main.rs

```diff
@@ -28,16 +28,16 @@ async fn main() -> Result<()> {
         None => soldr::Config::default(),
     };
 
-    let (ingest, mgmt, pool) = soldr::app(config).await?;
+    let (ingest, mgmt, pool, mgmt_listener, ingest_listner) = soldr::app(config).await?;
```

Comment:
because none of the tests are exercising this behavior, it hints to me that we should have read these configs directly.